### PR TITLE
Add post on TODO talks at CollabSummit

### PR DESCRIPTION
**Pull request by caniszczyk**
A blog post on TODO talks at the Collaboration Summit.

Signed-off-by: Chris Aniszczyk <caniszczyk@gmail.com>

### CODE REVIEW

File: _posts/2016-04-05-todo-track-at-collab-summit.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Listed below are some of slides from those talks:
+
+Gil Yehuda ([@gyehuda](https://twitter.com/gyehuda)) runs the Open Source Office at Yahoo and presented a talk highlighting some of the governance issues facing corporate based open source program offices. Gil made the case that medium to large tech companies needs an Open Source Program Office. The tech industry acknowledges the ubiquity of open source in the corporate tech stack as well as in products, yet companies face governance issues that need to be addressed properly, consistently, and with goals in mind. He noted that Jim Zemlin lamented at the event keynote that only about 30% of the tech companies surveyed had basic Open Source Governance policies in place, let alone a mature process to manage their involvement in Open Source. The industry needs to improve this, since corporate contributions to Open Source are an increasingly important part of the ecosystem.
```
Comment (DuaneOBrien) [inline _posts/2016-04-05-todo-track-at-collab-summit.md]: Bit of a nit-pick, but I think the post will improve if we're more consistent in office naming and in how we use caps on "Open Source". 

In this paragraph, Gil runs the Open Source Office and discusses the needs of an Open Source Program Office. In the next paragraph he details how an Open Source program office (program and office lower-cased) and has a thesis about an OSPO.

In the subsequent paragraphs, Brandon, Jeff and Christine run Open Source Offices for their respective companies. It's probably fine to describe all of us as running Open Source Offices for our respective companies, and I don't think it matters much what we all call them, but the flip back and forth from Open Source Office and Open Source Program Office and OSPO in the post feels inconsistent.

Same thing, but for instances of Open Source vs open source. (I say that, as a notorious over-user of Capital Letters for All The Things).

Comment (caniszczyk) [inline _posts/2016-04-05-todo-track-at-collab-summit.md]: Thanks, we will do a copy edit when we have slides posted.

Having a common lexicon amongst ourselves would be useful.

**Comment by caniszczyk** [comment]
I added Christine's talk, @jeffmcaffer, do you have your talk posted anywhere? I would love to add it

**Comment by nruff** [comment]
Hey Chris, while I did not attend, I did prepare slides and will send them next weekend. 
Nithya

**Pull request merged**